components/dashboards-web-component/src/viewer/components/gl-override-styles.css

```diff
@@ -29,3 +29,53 @@
     overflow: initial;
 }
 
+/* Side bar navigation styles */
```

Comment:
`gl-override-styles.css` file is for golden layout CSS overrrides (as the name says). 

Can we make these styles inline? If not, lets move them to a separate CSS file.

Reply:
Shall we have one CSS file per component ? and section accordingly. IMO it would be much cleaner than importing multiple CSS files. We can rename `gl-override-styles.css` to a more generic one.

Reply:
We went that path, and it resulted long, ugly, & bulky CSS files. Refer the [dashboard.css](https://github.com/wso2/carbon-dashboards/blob/master/components/dashboards-web-component/public/css/dashboard.css) file. (BTW we need to remove that). That is why we deviate from that and moved into inline styles and using CSS files when there is no other choice. A common CSS file defeats the purpose of component based architecture.

_An ideal solution will be creating a new component that renders a "Side Bar Nav" and that component have a CSS file._